src/chocolatey/docfx/docfx.svg

```diff
@@ -0,0 +1,10 @@
+<path d="M230 1359 c0 -18 11 -30 44 -48 80 -42 81 -45 81 -441 0 -400 -1 -404 -79 -436 -36 -15 -46 -24 -46 -43 0 -23 2 -24 61 -17 34 3 88 6 120 6 l59 0 0 495 0 495 -82 0 c-46 0 -100 3 -120 6 -35 6 -38 5 -38 -17z" style="border-bottom-color:rgb(255, 255, 255);border-left-color:rgb(255, 255, 255);border-right-color:rgb(255, 255, 255);border-top-color:rgb(255, 255, 255);color:rgb(255, 255, 255);cursor:pointer;height:auto;outline-color:rgb(255, 255, 255);overflow-x:visible;overflow-y:visible;width:auto;-webkit-column-rule-color:rgb(255, 255, 255);perspective-origin:0px 0px;-webkit-text-emphasis-color:rgb(255, 255, 255);-webkit-text-fill-color:rgb(255, 255, 255);-webkit-text-stroke-color:rgb(255, 255, 255);transform-origin:0px 0px;fill:rgb(221, 221, 221);"/>
+<path d="M618 1373 l-118 -4 0 -493 0 -494 154 -7 c181 -9 235 -3 313 34 68 33 168 130 207 202 75 136 75 384 1 536 -71 145 -234 240 -399 231 -23 -1 -94 -4 -158 -5z m287 -119 c68 -24 144 -101 176 -179 22 -54 24 -75 24 -210 0 -141 -2 -153 -26 -206 -36 -76 -89 -132 -152 -160 -45 -21 -68 -24 -164 -24 -71 0 -116 4 -123 11 -22 22 -31 175 -28 463 2 208 6 293 15 302 32 32 188 33 278 3z" style="border-bottom-color:rgb(255, 255, 255);border-left-color:rgb(255, 255, 255);border-right-color:rgb(255, 255, 255);border-top-color:rgb(255, 255, 255);color:rgb(255, 255, 255);cursor:pointer;height:auto;outline-color:rgb(255, 255, 255);overflow-x:visible;overflow-y:visible;width:auto;-webkit-column-rule-color:rgb(255, 255, 255);perspective-origin:0px 0px;-webkit-text-emphasis-color:rgb(255, 255, 255);-webkit-text-fill-color:rgb(255, 255, 255);-webkit-text-stroke-color:rgb(255, 255, 255);transform-origin:0px 0px;fill:rgb(221, 221, 221);"/>
+<path d="M1170 1228 c75 -104 110 -337 76 -508 -21 -100 -56 -178 -105 -233 l-36 -41 34 20 c75 43 160 133 198 212 37 75 38 78 38 191 -1 129 -18 191 -75 270 -28 38 -136 131 -153 131 -4 0 6 -19 23 -42z" style="border-bottom-color:rgb(255, 255, 255);border-left-color:rgb(255, 255, 255);border-right-color:rgb(255, 255, 255);border-top-color:rgb(255, 255, 255);color:rgb(255, 255, 255);cursor:pointer;height:auto;outline-color:rgb(255, 255, 255);overflow-x:visible;overflow-y:visible;width:auto;-webkit-column-rule-color:rgb(255, 255, 255);perspective-origin:0px 0px;-webkit-text-emphasis-color:rgb(255, 255, 255);-webkit-text-fill-color:rgb(255, 255, 255);-webkit-text-stroke-color:rgb(255, 255, 255);transform-origin:0px 0px;fill:rgb(221, 221, 221);"/>
+</g>
+</svg>
```

Comment:
This was gathered from the SVG at http://dotnet.github.io/docfx/. Once this is approved, we can repoint the icon url in the package to this one.